collective/easyform/profiles/default/registry.xml

```diff
@@ -0,0 +1,2 @@
+<records interface="collective.easyform.controlpanel.IEasyFormControlPanel"
```

Comment:
This file is missing start and end tags: `<?xml version="1.0"?><registry>YOURSTUFFGOESHERE</registry>`